haiku-apps/sidplayer/sidplayer-4.4.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A replayer program for C64 music (SID tunes) for Haiku and Unix"
+DESCRIPTION="SIDPlayer is a replayer program for C64 music (SID tunes) for BeOS \
+and Unix. You can think of it as being a stripped-down C64 emulator that only \
+emulates the processor and the sound chip of the C64. \
+SIDPlayer can be used as a stand-alone player or, under Haiku, as a plug-in \
+for CL-Amp and SoundPlay."
+HOMEPAGE="http://sidplayer.cebix.net/"
+COPYRIGHT="2003 Christian Bauer"
+LICENSE="GNU GPL v1"
+REVISION="1"
+SOURCE_URI="http://sidplayer.cebix.net/downloads/SIDPlayer-4.4.tar.gz"
+CHECKSUM_SHA256="fd1f73047abeb8234f850975786c805262678e77013835a8e621dcfd237624d7"
+SOURCE_FILENAME="SIDPlayer-$portVersion.tar.gz"
+SOURCE_DIR="SIDPlayer-$portVersion"
+
+ARCHITECTURES="x86_gcc2 !x86 !x86_64"
+
+PROVIDES="
+	sidplayer = $portVersion
+	cmd:sidplayer = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:make
+	cmd:autoreconf
+	cmd:aclocal
+	"
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure --omit-dirs binDir ./configure --bindir=$binDir
```

Comment:
Isn't --bindir already the standard option used by runConfigure?

Reply:
@korli I don't know.

Reply:
Then just use: "runConfigure ./configure"